lv_port/lv_port_display_espressif.c

```diff
@@ -0,0 +1,41 @@
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+
+#include "display_hal.h"
+
+#include "sdkconfig.h"
+#include "driver/gpio.h"
+
+void display_hal_delay(lv_disp_drv_t *drv, uint32_t delay_ms)
```

Comment:
I thought we agreed on `display_port_***` instead of `display_hal_***`
Applies to `display_hal.h` too

Reply:
We can change the name, will do later today.

Reply:
Fixed with d02d309